src/JSON For mIRC.mrc

```diff
@@ -763,7 +763,7 @@ alias JSON {
   }
 
   ;; Otherwise, do basic validation on the @Name parameter
-  elseif (: isin $1 || * isin $1 || ? isin $1) || ($1 == 0 && $0 !== 1) {
+  elseif (: isin $1) || (* isin $1) || (? isin $1) || ($1 === 0) && ($0 !== 1) {
```

Comment:
This is incorrect. the correct version would be the following but I choose not to include the extra `()`:

```
elseif ((: isin $1) || (* isin $1) || (? isin $1)) || (($1 == 0) && ($0 !== 1)) {
```